content/projects/gsoc/2023/index.adoc

```diff
@@ -1,4 +1,148 @@
 ---
-layout: redirect
-redirect_url: /projects/gsoc
+layout: project
+title: "Google Summer of Code in Jenkins"
+description: "Landing page for the Google Summer of Code program in the Jenkins project"
```

Comment:
Since this is for 2023, I think it would be better to include 2023 in the page description
```suggestion
description: "Google Summer of Code 2023 in the Jenkins project"
```